.github/workflows/tox.yaml

```diff
@@ -4,7 +4,7 @@ on: [pull_request]
 jobs:
```

Comment:
Nice to have: restrict the gh runner top-level permissions for the workflow like we do it in k8s-snap: https://github.com/canonical/k8s-snap/blob/main/.github/workflows/integration.yaml#L14

Reply:
Done!  nice